tests/ragger/test_name_version.py

```diff
@@ -0,0 +1,14 @@
+from ragger.backend import SpeculosBackend
+
+
+def test_get_app_and_version(cmd):
+    # for now, Speculos always returns app:1.33.7, so this test is not very meaningful
+    # however, its failure will mean that Speculos may have been improved on this front
+    app_name, version = cmd.get_app_and_version()
+
+    if not isinstance(cmd.client, SpeculosBackend):
+        assert app_name == "Boilerplate"
+        assert version == "1.0.1"
```

Comment:
No way to bind this from the Makefile version sadly ?

Reply:
There may be a way, but that could also be a reminder that the client itself may need to be updated too (if the app changed).

It's still better to update the version here than to have to update X screenshots.